Makefile

```diff
@@ -65,6 +69,8 @@ fixtures: fixtures/docker \
 	fixtures/drpm \
 	fixtures/drpm-signed \
 	fixtures/drpm-unsigned \
+	fixtures/file \
+	fixtures/file-bad \
```

Comment:
Good catch, I started with file-bad but since it has also valid entries decided to update to file-mixed which I think makes more sense. Will fix.